Xext/security.c

```diff
@@ -363,7 +363,7 @@ ProcSecurityQueryVersion(ClientPtr client)
         swaps(&rep.majorVersion);
         swaps(&rep.minorVersion);
     }
-    WriteToClient(client, SIZEOF(xSecurityQueryVersionReply), &rep);
+    X_SEND_REPLY_SIMPLE(client, rep);
```

Comment:
Did you forget to add the usual changes above or maybe they got into other PR somehow?

Reply:
indeed I did. thanks.